spring-data-vmware-gemfire-client/src/integrationTest/java/org/springframework/data/gemfire/GemfireTemplateIntegrationTests.java

```diff
@@ -175,15 +176,6 @@ public void containsKey() {
 		assertThat(this.usersTemplate.containsKey("dukeNukem")).isFalse();
 	}
 
-	@Test
-	public void containsKeyOnServer() {
-
-		assumeThat(CacheUtils.isClient(this.gemfireCache)).isTrue();
-
-		assertThat(this.usersTemplate.containsKeyOnServer(getKey(getUser("jackHandy")))).isTrue();
-		assertThat(this.usersTemplate.containsKeyOnServer("maxPayne")).isFalse();
-	}
-
```

Comment:
why was this test removed?

Reply:
This test was previously ignored due to the `assumeThat` because it was originally NOT a client cache. Now that I changed it to use a client cache, it runs and fails with `UnsupportedOperationException: Server keySet requires a pool.`. If I removed the `assumeThat` on develop, it fails the same way. So I removed it because it previously never ran and does not pass now or on develop.

Reply:
So this test is a server test?!?

Reply:
Not exactly. I see it as a bad test since it never ran before and wouldn't have passed if it had, either before or now. Not without otherwise changing the test.